_includes/js/custom.js

```diff
@@ -99,4 +69,383 @@ jQuery(document).ready(function() {
+                            </th>
+                            <th class="col-sm-3">Description</th>
+                            <th class="col-sm-4">Quick Access</th>
+                            <th class="col-sm-2">
+                                Re-Use
+                                <button type="button" class="btn btn-outline-default btn-sm" title="Sort by License" data-sort="license" style="float: right;">
+                                    <span aria-hidden="true" class="glyphicon glyphicon-chevron-down"></span>
+                                </button>
+                            </th>
+                            <th class="col-sm-1">Social</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        ${content}
+                    </tbody>
+                </table>
+                `;
+    }
+
+    /**
+     * converts json data passed in into renderTable into html table
+     * @param {object} data Ontology json data.
+     * @param {boolean} [domain=false]
+     */
+    function renderTable(data, domain= false ) {
+        let table = ``;
+        let domainTables = ``;
+        let tableDomains = []; // list of domains
+        let tableDomainhtml = {}; // hold html table data with domain as key
+        for (let i = 0; i < data.length; i++) {
+            let id = data[i]['id'];
+            let is_obsolete = "";
+            let is_inactive = "";
+            let activity_status = data[i]['activity_status']
+            let title = data[i]['title'];
+            let license_url = "#";
+            let license_logo = "#";
+            let license_label = "";
+            let description = data[i]['description'];
+            let repo = "";
+            let homepage = data[i]["homepage"];
+            let tracker = "";
+            let contact = "";
+            let publication = "";
+            let foundry_order = "";
+            let domainInner = ["Unknown"];
+
+            if (data[i]['license']) {
+                license_url = data[i]["license"]["url"];
+                license_logo = data[i]["license"]["logo"];
+                license_label = data[i]["license"]["label"];
+            }
+            if (data[i]["repository"] && data[i]["repository"].includes("https://github.com/")) {
+                repo = data[i]["repository"];
+            }
+            if (data[i]["tracker"]) {
+                tracker =`<a class="btn btn-default btn-sm" href="${data[i]["tracker"]}" aria-label="Go to the tracker for ${title}" title="Go to the tracker for ${title}">
+                                <span class="glyphicon glyphicon-comment" aria-hidden="true"></span>
+                            </a>`;
+            }
+            if (data[i].hasOwnProperty("domain") && data[i]['domain'] !== undefined) {
+                domainInner[0] = data[i]['domain'];
+            }
+            if (description !== undefined && description.toString().length > 140) {
```

Comment:
140 might be too long given this layout, some of the descriptions get up to 6 lines.

Reply:
This also needs a right-strip of spaces since the ellipses is applied inconsistently

Reply:
https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1994